nixos/modules/services/networking/webhook.nix

```
@@ -0,0 +1,163 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.webhook;
+  defaultUser = "webhook";
+
+  hooksFile = pkgs.writeText "webhook-config" cfg.hooks;
+
+in {
+  options = {
+    services.webhook = {
+      enable = mkEnableOption (mdDoc ''
+        [Webhook]. It's a server written in Go that allows you to create HTTP endpoints (hooks),
+        which execute configured commands for any person or service that knows the URL.
+
+        [Webhook]: https://github.com/adnanh/webhook
+      '');
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.webhook;
+        defaultText = literalExpression "pkgs.webhook";
+        example = literalExpression "pkgs.webhook";
+        description = mdDoc ''
+          Which Webhook package to use
+
+          This allows passing different version of the package.
+        '';
+      };
```

```suggestion
      package = mkPackageOption pkgs "webhook" {};
```

Sorry to come back to this a third time 😅